upload-src.sh

```diff
@@ -0,0 +1,19 @@
+# without sandbox please move creation of the bucket to terraform
+sh setup-src-bucket.sh
+
+mkdir build
+
+cd joke-data-crawler/src
+zip -r ../../build/joke-crawler.zip .
+cd ../..
+
+# "upload to s3"
+aws s3 cp build/joke-data-crawler.zip s3://joke-src-bucket-dzaa1417/
```

Comment:
there is a typo

use:
aws s3 cp build/joke-crawler.zip s3://joke-src-bucket-dzaa1417/